src/block.js

```diff
@@ -134,25 +134,13 @@ Block.calculateTarget = function (bits) {
 }
 
 Block.calculateMerkleRoot = function (transactions) {
-  var length = transactions.length
-  if (length === 0) throw TypeError('Cannot compute merkle root for zero transactions')
+  if (transactions.length === 0) throw TypeError('Cannot compute merkle root for zero transactions')
```

Comment:
I suppose @jprichardson we should decide what parts of the API should have type checking,  and what shouldn't.  Its definitely not consistent right now.